frontend/config/webpack.config.js

```diff
@@ -75,8 +75,7 @@ module.exports = {
       containers: path.resolve(srcPath, 'containers'),
       utils: path.resolve(srcPath, 'utils'),
       constants: path.resolve(srcPath, 'constants'),
-      connect: path.resolve(srcPath, 'base', 'connect'),
-      Container: path.resolve(srcPath, 'base', 'Container'),
```

Comment:
Don't forget to update the Container imports too

Reply:
I couldn't find any 🤷‍♂️

Reply:
😂 ok!!